nextcloudappstore/core/views.py

```diff
@@ -14,6 +15,9 @@ class AppDetailView(DetailView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['categories'] = Category.objects.all()
+        context['latest_releases_by_platform_v'] = \
+            self.object.latest_releases_by_platform_v()
+        context['desc_markdown'] = markdown(self.object.description)
```

Comment:
Still not fixed hm :/

We need to figure out how to prevent this or go for a JS solution which disallows that